CHANGELOG.md

```diff
@@ -17,6 +25,103 @@
+  Requires `track_functions=pl|all` in Postgres config
+- (Contrib) `postgresqlreceiver`: Add 'postgresql.temp.io' metric ([#41365](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/41365))
+  The metric emits the amount of data written to temporary files.
+- (Contrib) `kafkaexporter`: Add profiles support ([#41369](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/41369))
+- (Contrib) `prometheusreceiver`: Add retry logic for connection refused errors so the collector doesn't crash at startup. ([#40982](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/40982))
+  This change adds retry logic for connection refused errors. The target allocator could be
+  busy starting up the receiver and the first connection attempt may fail.
+
+- (Contrib) `receiver/prometheus`: Add support for otel_scope_schema_url label mapping to OpenTelemetry ScopeMetrics schema URL field ([#41488](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/41488))
+- (Contrib) `architecture`: New Tier 3 platform riscv64 allowing the collector to be built and distributed for this platform. ([#41507](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/41507))
+- (Contrib) `redactionprocessor`: The new `redact_all_types` option redacts all field types after casting them to string. ([#36684](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/36684))
+- (Contrib) `sqlserverreceiver`: Add 'sqlserver.computer.uptime' metric ([#41207](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/41207))
+- (Contrib) `sqlserverreceiver`: Make queries compatible with Azure SQL Database and Azure SQL Managed Instance. ([#41102](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/41102))
+- (Contrib) `telemetrygen`: Adds a new telemetrygen flag to generate unique timestamps for within a second interval ([#39933](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/39933))
+
+### 🧰 Bug fixes 🧰
+
+- (Core) `exporterhelper`: Prevents the exporter for being stuck when telemetry data is bigger than batch.max_size ([#12893](https://github.com/open-telemetry/opentelemetry-collector/issues/12893))
+- (Core) `mdatagen`: Fix import paths for mdatagen component ([#13069](https://github.com/open-telemetry/opentelemetry-collector/issues/13069))
+- (Core) `otlpreceiver`: Error handler correctly fallbacks to content type ([#13414](https://github.com/open-telemetry/opentelemetry-collector/pull/13414))
+- (Core) `pdata/pprofiles`: Fix profiles JSON unmarshal logic for originalPayload. The bytes have to be base64 encoded. ([#13483](https://github.com/open-telemetry/opentelemetry-collector/pull/13483))
+- (Core) `xpdata`: Fix unmarshaling JSON for entities, add e2e tests to avoid this in the future. ([#13480](https://github.com/open-telemetry/opentelemetry-collector/pull/13480))
```

Comment:
I would exclude this to reduce noise. There is no way users can send json otlp entity refs at this point.